app/routes/users/components/RemovePicture.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+
+import React, { Component } from 'react';
+import Button from 'app/components/Button';
+import Flex from 'app/components/Layout/Flex';
+
+import styles from './GroupChange.css';
+
+type Props = {
+  removePicture: string => Promise<*>,
+  username: string
+};
+
+type State = {
+  selected: boolean
+};
+
+export default class RemovePicture extends Component<Props, State> {
+  constructor() {
+    super();
+    this.state = {
+      selected: false
+    };
+  }
+
+  handleOnClick = () => {
+    if (this.state.selected)
+      this.props
+        .removePicture(this.props.username)
+        .then(() => this.setState({ selected: false }));
+  };
+
+  handleChange = () => {
+    this.setState({ selected: true });
+  };
+
+  handleReset = () => {
+    this.setState({ selected: false });
+  };
```

Comment:
A little picky, but this can just be one function
```js
toggleSelected = () => this.setState( { selected: !this.state.selected } )
```
Since the two buttons that use the functions are mutually exclusive there will never be the option of what button to press, meaning a choice never occurs. In these cases, the two functions can be merged.